app/Http/Middleware/CheckIfInitialized.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Middleware;
+use Closure;
+use Illuminate\Support\Facades\DB;
+
+
+
+class CheckIfInitialized
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
```

Comment:
```suggestion
     * @param Request $request
```
Same with other parameters' types